tools/meson-post-install.sh

```diff
@@ -27,6 +27,9 @@ for dir in "${pkgconfdir}/conf.d" \
            "${pkgconfdir}/dnsmasq.d" \
            "${pkgconfdir}/dnsmasq-shared.d" \
            "${pkglibdir}/conf.d" \
+           "${pkglibdir}/dispatcher.d/no-wait.d" \
+           "${pkglibdir}/dispatcher.d/pre-down.d" \
+           "${pkglibdir}/dispatcher.d/pre-up.d" \
```

Comment:
could you reorder this commit to be last, so that the intermediate commits of the branch still make sense?

Reply:
I think would break the RPM build which needs the directories.